learning_resources/serializers.py

```diff
@@ -375,5 +380,74 @@ class LearningPathRelationshipSerializer(LearningResourceRelationshipSerializer)
     """Specialized serializer for a LearningPath relationship"""
 
     relation_type = serializers.HiddenField(
-        default=LearningResourceRelationTypes.LEARNING_PATH_ITEMS.value
+        default=constants.LearningResourceRelationTypes.LEARNING_PATH_ITEMS.value
+    )
+
+
+class ContentFileSerializer(serializers.ModelSerializer):
```

Comment:
There are some fields not included here that will be required for opensearch (namely, `resource_relations` that indicates the parent-child relationship between the parent course and the contentfile). My thinking is that there will still be a `search.serializers.OSContentFileSerializer` that could inherit from this serializer and add that field.  Let me know if you'd prefer all the fields required for opensearch to be defined here.